prisma/schema.prisma

```diff
@@ -28,6 +28,8 @@ model Profile {
 model User {
   id      Int      @id @default(autoincrement())
   email   String   @unique
+  username String  @unique
```

Comment:
We are not storing any information of the user in local database so this schema is not needed